functions/index.js

```diff
@@ -15,22 +21,58 @@ const SCOPES = [
   'https://www.googleapis.com/auth/gmail.modify',
   'https://www.googleapis.com/auth/gmail.compose',
   'https://www.googleapis.com/auth/gmail.send',
+  'https://www.googleapis.com/auth/admin.directory.group',
+  'https://www.googleapis.com/auth/admin.directory.user',
+  'https://www.googleapis.com/auth/admin.directory.group.member',
+  'https://www.googleapis.com/auth/admin.directory.user.alias',
 ];
 
-// TODO: Change https request listener to firestore listener.
-exports.sendMail = functions.https.onRequest((request, response) => {
-  const recipient = request.query.recipient;
-  // TODO: Respond with more meaningful response.
-  response.send('Hello from Firebase!');
+exports.sendMail = functions.firestore
+    .document('pending-users')
```

Comment:
```suggestion
    .document('pending-users/{doc-id}')
```

Sorry, I forgot there is a need for wildcard here.